.github/workflows/create-sdk-update-pr.yml

```diff
@@ -0,0 +1,64 @@
+name: Create SDK update PR
+
+on:
+  workflow_dispatch:
+    inputs:
+      sdk_name:
+        description: 'The SDK for which the version should be updated'
+        required: true
+        type: choice
+        options:
+          - android
+          - ios
+      version_number:
+        description: 'The version number to which the SDK should be updated'
+        required: true
+        type: string
+
+jobs:
+  update:
+    runs-on: ubuntu-latest
+    env:
+      GH_TOKEN: ${{ github.token }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Setup Git User
+        run: |
+          git config --global user.name "Update Bot"
+          git config --global user.email "update-bot@bitmovin.com"
+
+      - name: Set Update Branch name
+        id: branching
+        run: |
+          branch_name="update_${{ inputs.sdk_name}}_player_to_${{ inputs.version_number }}"
```

Comment:
nit: Should we use `feature/` prefix?

Reply:
We have no convention in the repository. I would prefer to not use it. IMO there is no gain.
(I have to touch the file again soon. If you have a strong opinion, we can still change this 🙂)